scripts/tools/tools-object.js

```diff
@@ -93,17 +93,4 @@ async function convertTools(data) {
   return appendData;
 }
 
-function convertToJson(contentYAMLorJSON) {
 
-  //Axios handles conversion to JSON by default, if data returned for the server allows it
-  //So if returned content is not string (not YAML) we just return JSON back
-  if (typeof contentYAMLorJSON !== "string") return contentYAMLorJSON;
-
-  //in some cases json can be passed here as string as it failed parsing to json because of json related error
-  //instead of passint it to yaml parser, return same stuff that came in so it fails on JSON Schema validation later
-  if (contentYAMLorJSON.trimLeft().startsWith('{')) return contentYAMLorJSON
-
-  return yaml.parse(contentYAMLorJSON);
-}
-
-module.exports = { convertTools }
```

Comment:
@derberg I know these changes are done by mistake in this PR. Kindly copy the file contents from the master branch and paste them here so no conflicts are generated in this PR related to tools functionality.

Reply:
why do you think they are done by mistake? 😄 I moved this function to utils.js because I also use it in this PR